types/react-native-autocomplete-input/index.d.ts

```diff
@@ -75,15 +69,13 @@ export interface AutocompleteProps<T> extends TextInputProps {
      * function
      * render custom TextInput. All props passed to this function.
      */
-    renderTextInput?(props: TextInputProps): ReactNode;
+    renderTextInput?(props: AutocompleteProps<T>): ReactNode;
 
     /**
-     * string | bool
-     * Set `keyboardShouldPersistTaps` to true if RN version is <= 0.39.
+     * function
+     * render custom result list. Can be used to replace FlatList. All props passed to this function.
      */
-    renderSeparator?: ListViewProps['renderSeparator'] | undefined;
-
-    keyboardShouldPersistTaps?: 'always' | 'handeld' | 'never' | boolean;
```

Comment:
Removed `renderSeparator` and `keyboardShouldPersistTaps` from the props because these are not valid props for the component. They are not mentioned in the readme and all "extra" props get passed to `TextInput` ([see code](https://github.com/mrlaessig/react-native-autocomplete-input/blob/a011865598dd0b37c371892366f4212c18095735/index.js#L114)).

I believe these props were added because of a mistaken understanding of how the component works. The readme mentions setting `keyboardShouldPersistTaps` to fix an issue when putting the component inside of a `ScrollView`, but that prop is supposed to be added to `<ScrollView />`, not `<Autocomplete />`.